src/components/Task/MassActions/ConfirmationModal/MassActionsTasksConfirmationModal.tsx

```diff
@@ -20,23 +27,71 @@ export const MassActionsTasksConfirmationModal: React.FC<
 > = ({ open, idsCount, action, setOpen, onConfirm }) => {
   const { t } = useTranslation();
 
+  // KG: useState for checkbox
+  const [checked, setChecked] = useState(false);
```

Comment:
Can we make this variable more descriptive.

Having a name like `hasConfirmedDeletion` shows that it should have a boolean value by reading it, it also gives the reader an understanding of what it's used for.

```suggestion
  const [hasConfirmedDeletion, setHasConfirmedDeletion] = useState(false);
```